code/src/components/Adjective.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+const hornAdjectives = [
+  'golden',
+  'sparkling',
+  'shiny',
+  'colorful'
+]
+
+export const Adjective = ({ horn, setHorn }) => {
+  const handleHornAdjective = (event) => {
+    setHorn(event.target.value)
+  }
+  return (
+    <form className="horn-form" onSubmit={(event) => event.preventDefault()} required>
+      Choose a magical state:
+      {hornAdjectives.map((hornAdjective) => (
```

Comment:
It would have been a little easier to make a choice if the labels were clickable too.